code/modules/paperwork/fax.dm

```diff
@@ -255,6 +262,28 @@
 				loaded_item_ref = null
 				update_appearance()
 				return TRUE
+		
+		if("send_special")
+			if(!istype(loaded_item_ref?.resolve(), /obj/item/paper))
+				to_chat(usr, icon2html(src.icon, usr) + span_warning("Fax cannot send all above paper on this protected network, sorry."))
+				return 
+			
+			var/obj/item/paper/fax_paper = loaded_item_ref?.resolve()
+			fax_paper.request_state = TRUE
+			fax_paper.loc = null
```

Comment:
What is this doing and why are you choosing it over a qdel?

Reply:
give it ability to read everywhere, and send to null, so that it would be possible to read, but the object itself did not exist on the map

Reply:
When do the papers get deleted, then?

Reply:
In this example.. on server stop?
They hides, and you can read that paper again and again, from request panel.

Reply:
Ok thats probably fine given the context